SETUP.md

```diff
@@ -0,0 +1,32 @@
+# Installasjonsguide for workshop
+Guide for installasjon av editor, pakkebehandler og git.
+
+## Editor:
+Når det gjelder Editor står dere fritt til å velge selv, men f.eks VS Code er mye brukt blant frontendutviklere og lar deg installere støtte for Svelte.
```

Comment:
Forkortelser er mindre lesbare så de kan godt skrives helt ut: f.eks. -> for eksempel.